migration/rules/src/main/scala/fix/v0_9_28.scala

```diff
@@ -0,0 +1,22 @@
+package fix
+
+import scala.meta._
+
+import scalafix.v1._
+
+class v0_9_28 extends SemanticRule("v0_9_28") {
+  override def fix(implicit doc: SemanticDocument): Patch =
+    doc.tree.collect {
+      case t @ Importee.Name(Name("SemanticRuleSuite")) =>
+        Patch.removeImportee(t) +
+          Patch.addGlobalImport(
+            importer"scalafix.testkit.AbstractSemanticRuleSuite"
+          )
+      case t @ Init(_, _, _) =>
+        if (t.toString() == "SemanticRuleSuite()")
+          Patch.addGlobalImport(importer"org.scalatest.FunSuiteLike") +
```

Comment:
I just had a double-check on the mess that ScalaTest introduced in a minor release:
- `org.scalatest.FunSuiteLike` is available for ScalaTest < 3.2.0
- `org.scalatest.funsuite.AnyFunSuite` is available for ScalaTest >= 3.1.0

We currently build against 3.0.8, so the first is the right choice, and users of the testkit which have evicted 3.0.8 with 3.2.x would have runtime failures on `SemanticRuleSuite` anyway, so this migration does not apply to them.

We will easily be able to move to `org.scalatest.funsuite.AnyFunSuite` via another migration (at least where Steward runs) for a further version where we bump to 3.2.0 (or something else if we switch to munit).